libs/src/utils/utils.ts

```diff
@@ -1,7 +1,7 @@
 import bs58 from 'bs58'
 import Web3 from '../web3'
 import axios, { AxiosResponse } from 'axios'
-import Hashids from 'hashids/cjs'
+const Hashids = require('hashids/cjs')
```

Comment:
Did you try doing 

```ts
import Hashids from 'hashids'
```

Reply:
oh whoops, i just tried it and it works 👀!